runtime/default.policy

```diff
@@ -31,6 +58,53 @@ macros:
 # Rules determine what the Datadog security-agent will monitor.#
 ################################################################
 rules:
+## Process Activity Rules ##
+  - id: potential_web_shell
+    description: Potential webshell execution
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in HTTP_UTILS || exec.filename in SHELL_UTILS) && (process.ancestors.name in WEBAPP_PROCESSES || process.ancestors.name =~ "php*")
+  
+  - id: suspicious_database_process
+    description: Shell or shell utilites spawned by a database process
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in HTTP_UTILS || exec.filename in SHELL_UTILS) && process.ancestors.name in DATABASE_PROCESSES
+  
+  - id: suspicious_java_process
+    description: Shell or shell utilites spawned by a java process
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in HTTP_UTILS || exec.filename in SHELL_UTILS) && process.ancestors.name == "java"
+
+  - id: passwd_process_detected
+    description: passwd process detected
+    expression: >-
+      exec.filename == "/usr/bin/passwd"
+
+  - id: container_management_in_container
+    description: Docker client tool executed inside container
+    expression: >-
+      exec.filename in CONTAINER_PROCESSES && container.id != ""
+
+  - id: package_management_in_container
+    description: Package management detected in container
+    expression: >-
+      exec.filename in PKG_MAN && container.id != ""
+
+  - id: net_util_in_container
+    description: Network utility executed in container
+    expression: >-
+      (exec.filename in NET_UTILS || exec.filename in HTTP_UTILS) && container.id != ""
+
+  - id: net_util
+    description: Network utility executed
+    expression: >-
+      exec.filename in NET_UTILS
```

Comment:
We will want to keep an eye on this in testing to make sure that it doesn't have high noise levels